src/neo/Ledger/ContractState.cs

```diff
@@ -71,6 +72,30 @@ public JObject ToJson()
         {
             JObject json = new JObject();
             json["id"] = Id;
+
+            if (Id < 0)
+            {
+                var native = NativeContract.GetContract(Id);
+                if (native != null)
+                {
+                    json["name"] = native.Name;
+                    json["isnative"] = true;
+                }
+                else
+                {
+                    json["isnative"] = false;
+                }
+            }
+            else
+            {
+                if (Manifest.Name != null)
+                {
+                    json["name"] = Manifest.Name;
+                }
+
+                json["isnative"] = false;
+            }
+
             json["hash"] = ScriptHash.ToString();
             json["script"] = Convert.ToBase64String(Script);
             json["manifest"] = Manifest.ToJson();
```

Comment:
The name is included in the `manifest`, and I think we don't need to add it to the top level of the json.

Reply:
> The name is included in the `manifest`, and I think we don't need to add it to the top level of the json.

Agree. I'll close this.